CMakeLists.txt

```diff
@@ -33,9 +33,10 @@ project(hipcub LANGUAGES CXX)
 if(WIN32)
   set(ROCM_ROOT "$ENV{HIP_PATH}" CACHE PATH "Root directory of the ROCm installation")
 else()
-  set(ROCM_ROOT "/opt/rocm" CACHE PATH "Root directory of the ROCm installation")
+  set(ROCM_ROOT "/opt/rocm-6.0.1" CACHE PATH "Root directory of the ROCm installation")
```

Comment:
Is this something we want to change? Might better to keep this as `/opt/rocm` and only set version on the staging branch, that way we keep portability of the develop branch.

Reply:
Hi Nara, I will change this back, I just needed this on our local machine since it did not have rocm folder